datalake/src/main/resources/application.yml

```diff
@@ -68,6 +68,8 @@ altus:
   audit:
     enabled: true
     endpoint: localhost:8982
+  datalakedr:
+    endpoint: localhost:8982
```

Comment:
this port doesn't seem right to me, having the same value as audit

Reply:
8982 is the service port for all thunderhead services. What exactly is this file used for? I thought I had to add this setting here when I first did the change, but I'm also adding it to the Helm charts in [this review.](https://github.infra.cloudera.com/thunderhead/dps-k8s/pull/120/files), so I'm not sure when this setting is actually going to be used.

Reply:
so this is usually for default settings, as we try to avoid in code defaults. in case of endpoints, we can say the default is for local development. in that case this looks a good choice, to point this to the mock, but the mock doesn't have any implementation yet.
so I think we can delete the endpoint line as it doesn't make any difference as the client is disabled and there is nothing which it can call at that port

Reply:
Since we have `altus.datalakedr.enabled` set to `false` we don't' need to set any value to it.